dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22BeanValidationTests.java

```diff
@@ -26,7 +26,6 @@
 import com.gargoylesoftware.htmlunit.html.HtmlPage;
 import com.gargoylesoftware.htmlunit.html.HtmlTextInput;
 import com.ibm.websphere.simplicity.ShrinkHelper;
-import com.ibm.websphere.simplicity.config.ServerConfiguration;
```

Comment:
Please remember to update the copyright date in the header of any file which has functionally changed (any change except whitespace or formatting).

Reply:
Work on these files was in 2022, they already have a 2022 date in the header. I had to rebase and do a force push so it was a bit confusing. We are all set here.